hil/ext/obm/ipmi.py

```diff
@@ -68,10 +68,14 @@ def _ipmitool(self, args):
         return status
 
     @no_dry_run
-    def power_cycle(self):
+    def power_cycle(self, force=False):
         self._ipmitool(['chassis', 'bootdev', 'pxe'])
-        if self._ipmitool(['chassis', 'power', 'cycle']) == 0:
-            return
+        if force:
+            if self._ipmitool(['chassis', 'power', 'reset']) == 0:
+                return
+        else:
+            if self._ipmitool(['chassis', 'power', 'cycle']) == 0:
+                return
```

Comment:
Maybe factor out the differences here by using a variable:

```python
if force:
    op = 'reset'
else:
    op = 'cycle'
self._ipmitool(...)
...
```

Makes it easier to see what's going on.